packages/adp-tooling/templates/rta/fragment.xml

```diff
@@ -1,4 +1,7 @@
 <!-- Use stable and unique IDs!-->
-<core:FragmentDefinition xmlns:core='sap.ui.core' xmlns='sap.m'>
+<core:FragmentDefinition xmlns:core='sap.ui.core' xmlns='sap.m'><% if (targetAggregation && controlType) { %>
+    <!-- targetAggregation: <%= targetAggregation %> -->
+    <!-- controlType: <%= controlType %> --> <% } %>
```

Comment:
It is a minor thing, but shouldn't control type be before aggregation, because aggregation depends on the control type. The second one should be more readable.
```
targetAggregation: content
controlType: Toolbar
```

```
controlType: Toolbar
targetAggregation: content
```